e2e/ansible/roles/k8s-openebs-cleanup/tasks/main.yml

```diff
@@ -41,12 +41,18 @@
   delay: 30
   retries: 10
 
+- name: Delete the pre-release features
+  shell: source ~/.profile; kubectl delete cast -n {{ operator_ns }} --all
+  args:
+    executable: /bin/bash
+  register: cast
+
 - name: Confirm pre-release features have been deleted
   shell: source ~/.profile; kubectl get cast -n {{ operator_ns }}
   args:
     executable: /bin/bash
   register: result
-  until: "result.rc != 0 in result.stderr"
+  until: "result.rc == 0"
```

Comment:
@nsathyaseelan , rc will be zero even if the deletion failed in the above task. Please validate with 'no resources found' string.